### PR TITLE
Add Declarative Custom Elements write-up

### PR DESCRIPTION
**Pull request by Westbrook**
- add content from draft doc
- cc: @thescientist13 @bennypowers

### CODE REVIEW

File: reports/status.html [hunk 1]
```diff
@@ -57,6 +71,25 @@ <h2>Introduction</h2>
         </tbody>
       </table>
     </section>
+    <section>
+      <h2>Declarative custom elements</h2>
+      <dl>
+        <dt>GitHub Issue(s):</dt>
+        <dd>---</dd>
```
[the remark this answers is not in the document]
Comment (bennypowers) [reply reports/status.html:78]: https://github.com/WICG/webcomponents/blob/gh-pages/proposals/Declarative-Custom-Elements-Strawman.md

[the remark this answers is not in the document]
Comment (thescientist13) [reply reports/status.html:78]: @Westbrook 
Yeah, that strawman proposal that @bennypowers shared is the one I had been using in the spreadsheet, but hadn't found an actual issue so had neglected to put into our doc.  Do you think we should open a formal issue or do you think that proposal is sufficient?  If the latter, i can update our doc with it.

[the remark this answers is not in the document]
Comment (Westbrook) [reply reports/status.html:78]: I think this is fine for now.

I think we should open an official issue in some way. Whether that means just linking the explainer or starting something new listing the desires for such an as or suggesting alternate APIs as available, etc. having that stake in the ground would likely be a good way to prod this longer future conversation along a bit.

**Comment by bennypowers** [comment]
@Westbrook could i trouble you for a Co-Authored-By on the commit?

File: reports/status.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (thescientist13) [reply reports/status.html:78]: Sure, that sounds good.  Let me reach out to you 1:1 to coordinate something like that as I am happy to author / open an issue, but would have some questions to ask you and group first.

**Pull request merged**